packages/miniflare/src/workers/queues/broker.worker.ts

```diff
@@ -35,9 +35,9 @@ const DEFAULT_RETRIES = 2;
 
 const exceptionQueueResponse: FetcherQueueResult = {
 	outcome: "exception",
-	retryAll: false,
+	retryBatch: { retry: false, },
```

Comment:
Why did this need to be put in an object?

Reply:
this is the new `FetcherQueueResult` object shape

Reply:
the PR description points to the commit which introduced these changes: https://github.com/cloudflare/workerd/commit/3cc6fd9367fd5c2c79bf0ffabb1820b97c40bc1a